Primo.js

```diff
@@ -333,7 +353,7 @@ function importPNX(text) {
 	
 	item.edition = ZU.xpathText(doc, '//display/edition');
 	
-	var subjects = ZU.xpath(doc, '//search/subject');
+	var subjects = ZU.xpath(doc, '//display/display|//search/subject');
```

Comment:
Philipp Zumstein, 01/01/2015 23:15:
> I don't think that the xml-node |display| can have a child |display| again.
>

Thanks. This looks like a typo for "search"